documentation/blog/2023-09-20-runes.md

```diff
@@ -143,6 +151,8 @@ export function createCounter() {
 </button>
 ```
 
+> Runes can only be used in `.svelte.js` and `.svelte.ts` files.
```

Comment:
Part of Rich's (and my) suggestion here was that we should also mention `.svelte` files. It's confusing to say runes only work in `.svelte.js` and `.svelte.ts`.

Reply:
How about the new language? I didn't wanna put `.svelte`, '.svelte.js' and `.svelte.ts` all together, too many svelte's

Reply:
(It's already implied it works in .svelte files)